src/strand-datepicker/strand-datepicker.js

```diff
@@ -286,49 +286,50 @@
 					};
 				});
 
-				if(this._startUnix && this._endUnix) this.rangeValue = this._findRange(this._startUnix, this._endUnix);
+				if(this._startString && this._endString) this.rangeValue = this._findRange(this._startString, this._endString);
 			}
 		},
 
 		// Date bounds
-		_computeStartBound: function(start, allowedStart) {
-			var wrappedStart = _ensureMoment(start);
-			var wrappedAllowed = _ensureMoment(allowedStart);
-
-			if(wrappedStart.isValid() && wrappedAllowed.isValid()) {
-				return moment.max(wrappedStart, wrappedAllowed);
-			} else if(wrappedStart.isValid()) {
-				return wrappedStart;
-			} else {
-				return false;
+		_computeStartBound: function(startString, allowedStart) {
+			if(startString) {
+				var wrappedStart = _ensureMoment(startString);//.add(new Date().getTimezoneOffset(), 'minutes');
+				var wrappedAllowed = _ensureMoment(allowedStart);
+				if(wrappedStart.isValid() && wrappedAllowed.isValid()) {
+					return moment.max(wrappedStart, wrappedAllowed);
+				} else if(wrappedStart.isValid()) {
+					return wrappedStart;
+				} else {
+					return false;
+				}
 			}
 		},
-
-		_computeEndBound: function(end, allowedEnd) {
-			var wrappedEnd = _ensureMoment(end);
-			var wrappedAllowed = _ensureMoment(allowedEnd);
-
-			if(wrappedEnd.isValid() && wrappedAllowed.isValid()) {
-				return moment.min(wrappedEnd, wrappedAllowed);
-			} else if(wrappedEnd.isValid()) {
-				return wrappedEnd;
-			} else {
-				return false;
+		_computeEndBound: function(endString, allowedEnd) {
+			if(endString) {
+				var wrappedEnd = _ensureMoment(endString);//.add(new Date().getTimezoneOffset(), 'minutes');
+				var wrappedAllowed = _ensureMoment(allowedEnd);
+				if(wrappedEnd.isValid() && wrappedAllowed.isValid()) {
+					return moment.min(wrappedEnd, wrappedAllowed);
+				} else if(wrappedEnd.isValid()) {
+					return wrappedEnd;
+				} else {
+					return false;
+				}
 			}
 		},
 
 		_boundStart: function(newStart, oldStart) {
 			if(DataUtils.isDef(newStart) && newStart !== oldStart) {
-				this._startUnix = _clampDates(newStart, this.allowedStart, this._compositeAllowedEnd).unix();
-				if(!this._rangeValueFlag) this.rangeValue = this._findRange(this._startUnix, this._endUnix);
+				this._startString = _clampDates(newStart, this.allowedStart, this._compositeAllowedEnd).format();
```

Comment:
the format without considering dateFormat makes me nervous cause if the implementations in moment were ever to mismatch it would break the 2 instances of this